lib/led-matrix.cc

```diff
@@ -205,7 +205,14 @@ RGBMatrix::RGBMatrix(GPIO *io, const Options &options)
   case 4:
     ApplyStaticTransformer(internal::ZStripeTransformer(params_.rows * 2,
                                                         params_.cols / 2));
-    break;														
+    break;												
+  case 5:
+    ApplyStaticTransformer(internal::CoremanTransformer(params_.rows * 2,
+                                                          params_.cols / 2));
+    break;
+  case 6:
```

Comment:
I don't think you want to keep this one. This mention of the `UArrangmentTransformer` looks like the leftover from the previous change (Other, higher-level transformers that are not meant per panel are to be added somewhere else).